instruments/qubitekk/cc1.py

```diff
@@ -43,6 +43,7 @@ def __init__(self, filelike):
         self.terminator = "\n"
         self._channel_count = 3
         self._firmware = None
+        self._ack_on = False
```

Comment:
What happens in your older firmware if you send an invalid command? You might want to include straight up send `self.sendcmd(':ACKN OFF')` at `__init__` to make sure that your Python-state and firmware-state are in sync.